components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java

```diff
@@ -132,7 +132,7 @@ public static GenericArtifact getSampleAPIArtifactForTenant() throws GovernanceE
         artifact.setAttribute("overview_apiSecurity","oauth2,oauth_basic_auth_api_key_mandatory");
         artifact.setAttribute("URITemplate_authType0","Application & Application User");
         artifact.setAttribute("overview_keyManagers","[\"all\"]");
-        artifact.setAttribute("overview_environments","Production and Sandbox");
+        artifact.setAttribute("overview_environments","Default");
```

Comment:
remove this property itself.

Reply:
The underlying method and class is used only in test cases. So this attribute cannot be removed.